src/main/java/com/tagtraum/perf/gcviewer/model/AbstractGCEvent.java

```diff
@@ -24,6 +24,10 @@
     private int preUsed;
     /** Used after GC in KB */
     private int postUsed;
+    /** Percentage used before GC */
+    private int preUsedPercent;
```

Comment:
Is it really worthwhile to store this information? Can't it be calculated, if needed?

Reply:
You are right, I don't think it's worthwhile to have this, I will remove this.